documentation/modules/installing-mtv-operator.adoc

```diff
@@ -34,13 +34,14 @@ The {operator-name} is a Community Operator. Red Hat does not support Community
 ====
 endif::[]
 . Click *{operator-name-ui}* and then click *Install*.
-. On the *Install Operator* page, click *Install*.
-. Click *Operators* -> *Installed Operators* to verify that *{operator-name-ui}* appears in the *{namespace}* project with the status *Succeeded*.
-. Click *{operator-name-ui}*.
-. Under *Provided APIs*, locate the *ForkliftController*, and click *Create Instance*.
+. Click *Create ForkLift Controller* when the button becomes active.
```

Comment:
ForkLift should be Forklift, and there is no space between Forklift and Controller, so the button text should be *Create ForkliftController*